prow/config.yaml

```diff
@@ -2,6 +2,20 @@ plank:
   job_url_template: 'https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/{{if eq .Spec.Type "presubmit"}}pr-logs/pull{{else if eq .Spec.Type "batch"}}pr-logs/pull{{else}}logs{{end}}{{if .Spec.Refs}}{{if ne .Spec.Refs.Org ""}}{{if ne .Spec.Refs.Org "kubernetes"}}/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}{{else if ne .Spec.Refs.Repo "kubernetes"}}/{{.Spec.Refs.Repo}}{{end}}{{end}}{{end}}{{if eq .Spec.Type "presubmit"}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else if eq .Spec.Type "batch"}}/batch{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
   report_template: '[Full PR test history](https://k8s-gubernator.appspot.com/pr/{{if eq .Spec.Refs.Org "kubernetes"}}{{if eq .Spec.Refs.Repo "kubernetes"}}{{else}}{{.Spec.Refs.Repo}}/{{end}}{{else}}{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{end}}{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}). [Your PR dashboard](https://k8s-gubernator.appspot.com/pr/{{with index .Spec.Refs.Pulls 0}}{{.Author}}{{end}}). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/{{.Spec.Refs.Org}}/{{.Spec.Refs.Repo}}/issues?q=is:issue+is:open) when you hit one in your PR.'
   pod_pending_timeout: 60m
+  default_decoration_config:
+    timeout: 7200000000000
```

Comment:
is this actually more readable than a time.Duration string like `120m`?

Reply:
I'm really not super interested in going _back_ to a string field that gets parsed at config load time and then stored in some other field for later use. We need external and internal config and we need conversions between them. I was asked to switch this the other way, so it is now a serialized `time.Duration`. I imagine it may be a tiny bit annoying but ultimately we will fix config soon enough and this will not be a field that gets touched very often so the pain is manageable in the meanwhile.

Reply:
can we generate a comment then at least? configs that are not easy for humans to read quickly are error prone. 

I don't think human unfriendly config should be config at all, and this file is definitely "user" facing.

Reply:
Agreed, this isn't pretty. I'll add comments.